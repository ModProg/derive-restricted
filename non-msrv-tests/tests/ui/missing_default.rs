#[derive_where::derive_where(Default; T)]
enum Test<T> {
	A(T),
}

fn main() {}
