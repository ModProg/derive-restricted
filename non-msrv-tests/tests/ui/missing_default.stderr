error: `enum`s require specifying a `default` option to implement `Default`
 --> tests/ui/missing_default.rs:2:1
  |
2 | / enum Test<T> {
3 | |     A(T),
4 | | }
  | |_^
